### ISSUE TITLE
Failed to output anytime other than the Quit button, after refreshing for 5 secs

### ISSUE THREAD

**Issue by VansonLeung** [jontelang/WakaLet#19]
<img width="181" alt="2015-11-13 4 03 18" src="https://cloud.githubusercontent.com/assets/1129695/11141415/1d03af44-8a20-11e5-9fff-6b8c12ddc00f.png">
When starting the wakalet app, clicking the menu dock icon should show "refreshing...", and after a while should show all my projects.

Instead it appears empty.

OSX Yosemite 10.0.5, Wakalet ver 0.2

**Comment by alexsteinerde** [comment]
I have the same problem.
OSX El Capitan 10.11.1, Wakalet ver 0.2

**Comment by 2Grey** [comment]
FIX
Replace in AppDelegate.m 171 line:
From:

``` objectivec
NSString* date_human = [rangeObj valueForKey:@"date_human"];
```

To:

``` objectivec
NSString* date_human = [rangeObj valueForKey:@"text"];
```

**Comment by henry930** [comment]
Thanks!

**Comment by jontelang** [comment]
@2Grey Thanks a lot! Fixed in master branch and in release 0.3.
